lib/constants.js

```diff
@@ -201,5 +201,15 @@ module.exports = Object.freeze({
     INACTIVE: 'inactive',
     DEPRECATED: 'deprecated'
   },
-  BASE64_ENC_NULL: 'bnVsbA=='
+  BASE64_ENC_NULL: 'bnVsbA==',
+  SERVICE_LIFE_CYCLE: {
+    PRE_CREATE: 'PreCreate',
+    POST_CREATE: 'PostCreate',
+    PRE_DELETE: 'PreDelete',
```

Comment:
I thought you were going to add in for even bind/unbind and even postupdate. See these 3 as missing.

Reply:
Added PreBind, PreUnbind. As discussed, not implementing any post hooks.